service/src/main/java/tds/assessment/repositories/impl/AdminSubjectQueryRepositoryImpl.java

```diff
@@ -44,7 +45,8 @@ public AdminSubjectQueryRepositoryImpl(final DataSource dataSource) {
                     rs.getString("key"),
                     rs.getString("assessmentId"),
                     rs.getBoolean("issegmented"),
-                    rs.getString("selectionalgorithm")));
+                    rs.getString("selectionalgorithm"),
+                    rs.getFloat("startAbility")));
```

Comment:
Do we want this to be null if the value in the DB is null?  If so, this won't work because `rs.getFloat()` returns 0 if the column value is null (docs [here](https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#getFloat(java.lang.String))).  If null is important I think we should leverage the solution mentioned (here)[http://stackoverflow.com/questions/2920364/checking-for-a-null-int-value-from-a-java-resultset]